ngx_rtmp_stat_module.c

```diff
@@ -343,7 +344,8 @@ ngx_rtmp_stat_get_pool_size(ngx_pool_t *pool, ngx_uint_t *nlarge,
 
 
 static void
-ngx_rtmp_stat_dump_pool(ngx_http_request_t *r, ngx_chain_t ***lll,
+ngx_rtmp_stat_dump_pool(ngx_rtmp_stat_loc_conf_t *slcf,
```

Comment:
`slcf` pointer can be got via `r` and it's easy to get it, so no need to add a new variable.

Reply:
Sorry for my fault. Fixed.